packages/core/crates/core-network/src/network.rs

```diff
@@ -5,7 +5,7 @@ use std::time::Duration;
 use libp2p::PeerId;
 
 use utils_log::{info,warn,error};
-use utils_metrics::metrics::native::{MultiGauge, SimpleGauge};
+use utils_metrics::metrics::{MultiGauge, SimpleGauge};
```

Comment:
So explicitly using the wasm version?

Reply:
There will be only a single version of all the metric objects and the distinction between Wasm and non-wasm is only in 2 functions in Histograms, where it is hidden via macro.